GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
Why remove this file? I think we need to include it and set next-version to 3.0.0